x/group/errors/errors.go

```diff
@@ -0,0 +1,19 @@
+package errors
```

Comment:
i have a slight preference for keeping it in `x/group`, so that we don't need to do `grouperrors "github.com/x/group/errors"`. But not blocking on this.

Reply:
this is for avoiding import cycle (group <-> orm), see https://github.com/cosmos/cosmos-sdk/pull/10451#discussion_r741180638